metal/v1/api_organizations.go

```diff
@@ -214,6 +215,12 @@ func (r ApiCreateOrganizationInvitationRequest) InvitationInput(invitationInput
 	return r
 }
 
+// Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects.
+func (r ApiCreateOrganizationInvitationRequest) Include(include []string) ApiCreateOrganizationInvitationRequest {
```

Comment:
Fixes #118: adds support for `include` parameter for the invitation create endpoint (`exclude` is not supported for that endpoint)